### PR TITLE
Codegen angular

### PR DESCRIPTION
**Pull request by yuth**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-codegen/src/utils/getFrontEndFramework.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const getFrontendHandler = require('./getFrontEndHandler');
+
+function getFrontEndFramework(context) {
```
Comment (kaustavghosh06) [inline packages/amplify-codegen/src/utils/getFrontEndFramework.js]: Shouldn't it just be getFrontendJavascript framework? Seems like very specific to javascript.

File: packages/amplify-codegen/src/utils/getOutputFileName.js [hunk 2]
```diff
@@ -1,18 +1,27 @@
 const path = require('path');
 
-function getOutputFileName(inputFileName = 'API', target) {
+function getOutputFileName(inputFileName, target) {
   const extensionMap = {
     swift: 'swift',
     typescript: 'ts',
     flow: 'js',
+    angular: 'service.ts',
```
Comment (kaustavghosh06) [inline packages/amplify-codegen/src/utils/getOutputFileName.js]: Is this the extension or the filename? the values for swift/typescript/flow etc looks like just extensions, unlike that for angular

File: packages/amplify-codegen/src/utils/getOutputFileName.js [hunk 3]
```diff
@@ -1,18 +1,27 @@
...
   };
+  const folderMap = {
+    javascript: 'src',
+    typescript: 'src',
+    flow: 'src',
+    angular: 'src/app',
+  };
+
+  inputFileName = inputFileName || target === 'angular' ? 'api' : 'API';
```
Comment (kaustavghosh06) [inline packages/amplify-codegen/src/utils/getOutputFileName.js]: Can you explain what this is doing? and in what cases target = api vs API? This check seems a bit fragile.

File: packages/amplify-graphql-types-generator/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
+# Apollo Contributor Guide
```
Comment (kaustavghosh06) [inline packages/amplify-graphql-types-generator/CONTRIBUTING.md]: Should remove this file? Looks like apollo contributor guide. We have a contributing block in our main readme, so we can get rid of this.

File: packages/amplify-graphql-types-generator/LICENSE [hunk 5]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```
Comment (kaustavghosh06) [inline packages/amplify-graphql-types-generator/LICENSE]: Remove this file. We have a License file on our mono repo whihc is Apcha Software and not MIT.

File: packages/amplify-graphql-types-generator/NOTICE [hunk 6]
```diff
@@ -0,0 +1,13 @@
+AWS AppSync Codegen for iOS
```
Comment (kaustavghosh06) [inline packages/amplify-graphql-types-generator/NOTICE]: Can get rid fo this as well.

File: packages/amplify-graphql-types-generator/README.md [hunk 7]
```diff
@@ -0,0 +1,226 @@
+
+
+
+
+# Amplify GraphQL types generator
```
Comment (kaustavghosh06) [inline packages/amplify-graphql-types-generator/README.md]: Is this file up to date with the latest changes to the cli?

File: packages/amplify-codegen/src/utils/getOutputFileName.js (hunk 2 above)
Comment (yuth) [reply packages/amplify-codegen/src/utils/getOutputFileName.js]: Yes. Angular adds `service.ts` to Services

**Comment by kaustavghosh06** [comment]
The licensing and author info should also be consistent with the other packages in the repo:

```
  "author": "Amazon Web Services",
  "license": "Apache-2.0",
```

File: packages/amplify-codegen/src/utils/getFrontEndFramework.js (hunk 1 above)
Comment (yuth) [reply packages/amplify-codegen/src/utils/getFrontEndFramework.js]: I don't think this code really belongs here. Everything should move to frontend handler eventually. But in the interest of time I am keeping it here

File: packages/amplify-codegen/src/utils/getOutputFileName.js (hunk 3 above)
Comment (yuth) [reply packages/amplify-codegen/src/utils/getOutputFileName.js]: For every other language other then angular we are using `API` as the base file name. For angular its lowercase. Also the first check is if the user has already specified the `inputFileName` we provide no default file name

File: packages/amplify-graphql-types-generator/LICENSE (hunk 5 above)
Comment (yuth) [reply packages/amplify-graphql-types-generator/LICENSE]: Are these 2 compatible with one another?

File: packages/amplify-graphql-types-generator/NOTICE (hunk 6 above)
Comment (yuth) [reply packages/amplify-graphql-types-generator/NOTICE]: OK

File: packages/amplify-graphql-types-generator/README.md (hunk 7 above)
Comment (yuth) [reply packages/amplify-graphql-types-generator/README.md]: I think I can get rid of this all together

**Comment by kaustavghosh06** [comment]
Ignore the licensing comments. Should retain the MIT license.

File: packages/amplify-graphql-types-generator/LICENSE (hunk 5 above)
Comment (yuth) [reply packages/amplify-graphql-types-generator/LICENSE]: Removed

**Pull request merged**